lessons/requirements.md

```diff
@@ -1,24 +1,27 @@
 ## Requirements
 
-This codelab is useful for anyone who are getting started with developing Firefly apps.
-
-### Required skills
-
-* **Required:** programming skills in any language
-* *Optional:* basic knowledge with Javascript and Node.js
-
 ### Pre-requisites
+This codelab is for anyone who are getting started with developing Firefly apps. There are a few prerequisites to get you started
+
+* Project Firefly app
 
-If not already done, you should follow the below links to have your environment set up before starting the codelab:
+This lab will be focus on Adobe I/O Events lib, for how to bootstrap Firefly app please see below or other codelab contents
+we woud assume that you already have an app developed and would like to leverage I/O event to build an event-driven flow
+  * [How to Get Access to Project Firefly](https://github.com/AdobeDocs/project-firefly/blob/master/overview/getting_access.md)
+  * [Setting up Your Environment](https://github.com/AdobeDocs/project-firefly/blob/master/getting_started/setup.md)
+  * [Creating your First Firefly App](https://github.com/AdobeDocs/project-firefly/blob/master/getting_started/first_app.md)
+  
+* Install Adobe I/O Events lib
```

Comment:
No need to install Events lib. Instead you should do `npm install -g @adobe/aio-cli-plugin-events`

Reply:
good catch, fixed it